src/components/common/view/Text.js

```diff
@@ -54,7 +54,7 @@ const relatedLineSpacing = fontSize =>
     24: 30,
     36: 30,
     42: 30,
```

Comment:
as far as I remember, we discussed to undo this change I did and set `lineHeight` with `normalize`, wasn't it?

Reply:
we are using lineHeight with normalize, on line 80...
```lineHeight: normalize(calculatedLineHeight),```

Reply:
oh... indeed :man_facepalming: